Samples/aggregate-traffic-count-data-within-localauthorities.py

```diff
@@ -1,22 +1,38 @@
+'''
+  This recipe gets two datasources
+     - London Traffic data using TrafficCountImporter.java only for London
+     - London localAuthority data using OaImporter.java
+  
+  Then for each localAuthority using GeographicAggregationField
+    - It sums all the bi-cycle count based on LocalAuthority
+  
+  Then it writes the sum to the output file for every localAuthority using GeographicAggregationField
```

Comment:
Maybe rephrase this to just 'it then sums the individual bicycle counts that are contained within a local aurhority using GeographicAggregationField'

Reply:
done